src/main/resources/portal.properties.EXAMPLE

```diff
@@ -234,7 +234,7 @@ oncoprint.hotspots.default=true
 # to enable use of Ehcache, set ehcache.cache_enabled to true and uncomment all Ehcache properties
 ehcache.cache_enabled=false
```

Comment:
we dont need this anymore right?

Reply:
Sorry we hadn't added that file to the commit.  Thanks!